contracts/src/TeleporterTokenDestination.sol

```diff
@@ -130,6 +123,18 @@ abstract contract TeleporterTokenDestination is
         return value / tokenMultiplier;
     }
 
+    /**
+     * @dev Calculates the number of 32-byte words required to fit a payload of a given length.
+     * The payloads are padded to have a length that is a multiple of 32.
+     */
+    function calculateNumWords(uint256 payloadSize) public pure returns (uint256) {
+        uint256 numWords = payloadSize / 32;
```

Comment:
Are bitshifts and masks any cheaper in gas than divides and mods? This would be a simple change if there is a benefit.

Reply:
Great call! Updated in #97